deploy/helm/sumologic/values.yaml

```diff
@@ -1948,6 +1948,17 @@ kube-prometheus-stack:
             - action: keep
               regex: (?:kafka_(broker_.*|controller_.*|java_lang_.*|partition_.*|purgatory_.*|network_.*|replica_.*|request_.*|topic_.*|topics_.*|zookeeper_.*))
               sourceLabels: [__name__]
+      
+        # MySQL Telegraf Metrics
+        # List of Metrics are on following github page:
+        # https://github.com/influxdata/telegraf/tree/master/plugins/inputs/mysql#metrics
+        # Metrics follow following format:
+        # mysql_*
```

Comment:
Please keep this comment consistent with what is in the regex. You can take a look at other remote write sections for example of how it can be done.

Reply:
Done